src/Modules/Engine.cpp

```diff
@@ -648,6 +652,13 @@ void _Host_RunFrame_Render_Detour();
 void (*_Host_RunFrame_Render)();
 static Hook _Host_RunFrame_Render_Hook(&_Host_RunFrame_Render_Detour);
 void _Host_RunFrame_Render_Detour() {
+	g_total_frames++;
+
+	if (now != time(NULL)) {
+		now = time(NULL);
+		g_cur_fps = (float)g_total_frames;
```

Comment:
this approach only allows us to update the fps once per second, which kinda sucks. a rolling average as originally explained by me on discord would be much better